Engines/Wine/QuickScript/GoG Script/script.js

```diff
@@ -0,0 +1,70 @@
+include(["engines", "wine", "quick_script", "quick_script"]);
+include(["utils", "functions", "net", "download"]);
+include(["engines", "wine", "engine", "object"]);
+
+function GogScript() {
+    QuickScript.call(this);
+}
+
+GogScript.prototype = Object.create(QuickScript.prototype);
+
+GogScript.prototype.constructor = GogScript;
```

Comment:
Why don't you add the constructor function here directly?

Reply:
I just did the same we do for other quick scripts